modules/python/src2/gen2.py

```diff
@@ -209,7 +209,8 @@ class FormatStrings:
     "int": ArgTypeInfo("int", FormatStrings.int, "0", True),
     "float": ArgTypeInfo("float", FormatStrings.float, "0.f", True),
     "double": ArgTypeInfo("double", FormatStrings.double, "0", True),
-    "c_string": ArgTypeInfo("char*", FormatStrings.string, '(char*)""')
+    "c_string": ArgTypeInfo("char*", FormatStrings.string, '(char*)""'),
+    "UMat": ArgTypeInfo("UMat", FormatStrings.object, 'UMat()', True),  # switch to CV_EXPORTS_W_SIMPLE
```

Comment:
Shouldn't it be a separate change, because `UMat` export strategy is not related to `FileStorage`

Reply:
It is a "TODO"/"FIXIT" comment.
There is no change for UMat at this time. It is a "repair" entry.